src/components/UnsplashImageFetcher.jsx

```diff
@@ -0,0 +1,40 @@
+import { useState, useEffect } from 'react';
+
+function UnsplashImageFetcher() {
+  const [error, setError] = useState('');
+
+  async function getPicsUnsplash() {
+    const url = `https://api.unsplash.com/photos/random?query=nature&client_id=${import.meta.env.VITE_UNSPLASH_API_KEY}`;
+
+    try {
+      const res = await fetch(url);
+      const data = await res.json();
+
+      if (res.status === 403) {
+        throw new Error('Failed to fetch photo from Unsplash API.');
+      } else if (res.status === 404) {
+        throw new Error('No photos found.');
+      }
```

Comment:
Consider what you'd like to happen if this fetch fails.  Maybe a static fallback background image that you include in the repo?  Right now it's just white.

Not a requirement for now, just food for thought.